api/api.tsx

```diff
@@ -82,6 +83,7 @@ const api = async (
     ok: response?.ok ?? false,
     clientError: response && response.status >= 400 && response.status < 500,
     json: json,
+    status: response?.status
```

Comment:
```suggestion
    status: response?.status ?? 0
```

Otherwise the type in `ApiResponse` should be `number | undefined`.

A nice beginner ticket would be to add type checks to CI